README.md

```diff
@@ -5,7 +5,7 @@
 |[15.8.x](docs/repo/roadmap.md)|[dev15.8.x](https://github.com/dotnet/project-system/tree/dev15.8.x)|[![Build Status](https://ci.dot.net/job/dotnet_project-system/job/dev15.8.x/job/windows_debug/badge/icon)](https://ci.dot.net/job/dotnet_project-system/job/dev15.8.x/job/windows_debug/)|[![Build Status](https://ci.dot.net/job/dotnet_project-system/job/dev15.8.x/job/windows_release/badge/icon)](https://ci.dot.net/job/dotnet_project-system/job/dev15.8.x/job/windows_release/)|[![codecov](https://codecov.io/gh/dotnet/project-system/branch/dev15.8.x/graph/badge.svg)](https://codecov.io/gh/dotnet/project-system/branch/dev15.8.x)
 |[15.9.x](docs/repo/roadmap.md)|[dev15.9.x](https://github.com/dotnet/project-system/tree/dev15.9.x)|[![Build Status](https://ci.dot.net/job/dotnet_project-system/job/dev15.9.x/job/windows_debug/badge/icon)](https://ci.dot.net/job/dotnet_project-system/job/dev15.9.x/job/windows_debug/)|[![Build Status](https://ci.dot.net/job/dotnet_project-system/job/dev15.9.x/job/windows_release/badge/icon)](https://ci.dot.net/job/dotnet_project-system/job/dev15.9.x/job/windows_release/)|[![codecov](https://codecov.io/gh/dotnet/project-system/branch/dev15.9.x/graph/badge.svg)](https://codecov.io/gh/dotnet/project-system/branch/dev15.9.x)
-|[16.0](docs/repo/roadmap.md)|[master](https://github.com/dotnet/project-system/tree/master)|[![Build Status](https://ci.dot.net/job/dotnet_project-system/job/master/job/windows_debug/badge/icon)](https://ci.dot.net/job/dotnet_project-system/job/master/job/windows_debug/) |[![Build Status](https://ci.dot.net/job/dotnet_project-system/job/master/job/windows_release/badge/icon)](https://ci.dot.net/job/dotnet_project-system/job/master/job/windows_release/)|[![codecov](https://codecov.io/gh/dotnet/project-system/branch/master/graph/badge.svg)](https://codecov.io/gh/dotnet/project-system)
+|[16.0](docs/repo/roadmap.md)|[master](https://github.com/dotnet/project-system/tree/master)|[![Build Status](https://ci.dot.net/job/dotnet_project-system/job/master/job/windows_debug/badge/icon)](https://ci.dot.net/job/dotnet_project-system/job/master/job/windows_debug/) |[![Build Status](https://ci.dot.net/job/dotnet_project-system/job/master/job/windows_release/badge/icon)](https://ci.dot.net/job/dotnet_project-system/job/master/job/windows_release/)|[![codecov](https://codecov.io/gh/dotnet/project-system/branch/master/graph/badge.svg)](https://codecov.io/gh/dotnet/project-system) [![Join the chat at https://gitter.im/dotnet/project-system](https://badges.gitter.im/dotnet/project-system.svg)](https://gitter.im/dotnet/project-system?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
This is off - it needs a new line before the link to not show up in the unit test table.